### PR TITLE
JSON loader

### PR DESCRIPTION
**Pull request by jenrik**

### CODE REVIEW

**Review by cogitantium — APPROVED** [review]
LGTM, I think we should hold off merging this until parser and logger is merged, such that this can be integrated in a sane way.

Mexican-standoff.json is bit-for-bit equal to the generated mexican standoff made by https://github.com/d702e20/encode-game-structure

**Review by MathiasMehl — APPROVED** [review]

**Comment by jenrik** [comment]
Merging since the prerequisite #14 have been merged

**Pull request merged**
